### ISSUE TITLE
Provide a throwing alternative to InspectableView subscript

### ISSUE THREAD

**Issue by tarbayev** [nalexn/ViewInspector#136]
Currently it's not possible to verify optional views, as there's no public **throwing** accessor for `ClassifiedView` type at a given index.

I'd like to avoid overspecialization of my tests and just check the actual number of children in a container, e.g. `ZStack`, or if there is a view at a particular index.

In addition would be nice if the `count` property returns the actual number of views in a container excluding invisible optional views.

## Example

```swift
struct MyView: View {
  let showsSecondView: Bool
  var body: some View {
    ZStack {
      Text("First")
      if showsSecondView {
        Text("Second")
      }
    }
  }
}
```

```swift
let myView = MyView(showsSecondView: false)
XCTAssertThrowsError(try myView.inspect().zStack()[1]) // Crashes instead of throwing
```

**Comment by nalexn** [comment]
I've made so that subscript does not crash for an absent view, and introduced a `var isAbsent: Bool` which can be used for asserting the view is not present:

```swift
XCTAssertTrue(try myView.inspect().zStack()[1].isAbsent)
```

That being said, the `count` still returns the total number of views, including the invisible ones (changing the semantics would break tests for many users), so instead, if you need to get to know the number of visible views, you can do `.filter({ !$0.isAbsent }).count`

**Comment by nalexn** [comment]
Released with v0.9.1
